### PR TITLE
Km 2 vet additions

### PR DESCRIPTION
**Pull request by OAKevin**
Admin page, with Vet creation page, Adding color changes to add a bit of color back to the background. Definitely has merge conflicts.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -218,6 +219,7 @@ const VetNavigator = createBottomTabNavigator(
         tabBarComponent: VetTab,
         tabBarPosition: "top",
         swipeEnabled: false,
+
```
Comment (Danny393) [inline App.js]: Nit pick, remove extra white space

File: src/components/Button.js [hunk 2]
```diff
@@ -44,16 +44,16 @@ export default class Button extends React.Component<> {
                 full = {full}
                 disabled = {disabled}
                 transparent = {!primary || transparent}
-            >
+
```
Comment (Danny393) [inline src/components/Button.js]: This page is causing issues and crashing the app

File: src/components/Fields.js [hunk 3]
```diff
@@ -43,5 +43,6 @@ const styles = StyleSheet.create({
         shadowOpacity: 0.03,
         shadowRadius: 1,
         width: "100%"
+
```
Comment (Danny393) [inline src/components/Fields.js]: Nit pick, remove extra white space

File: src/components/NavHeaderWithButton.js [hunk 4]
```diff
@@ -72,7 +72,7 @@ const styles = StyleSheet.create({
         borderColor: Theme.palette.borderColor,
         borderBottomWidth: Platform.OS === "ios" ? 0 : 1,
         zIndex: 10000,
-        backgroundColor: "white",
+        backgroundColor: "#f3f6f4",
```
Comment (Danny393) [inline src/components/NavHeaderWithButton.js]: We should not have hard code color values, see line 72 to use colors defined on Theme. Make sure to replace all other hard coded color values through out this PR

File: src/sign-up/SignUpVet.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+        justifyContent: "space-between",
+        justifyContent: "flex-start",
+        flex: .3,
+        paddingVertical:60,
+        
+    },
+    secondrow: {
+        margin:10,
+        
+        //alignSelf: "center",
```
Comment (Danny393) [inline src/sign-up/SignUpVet.js]: Clean up comments on this page, remove any that are no longer necessary

File: src/vet/patients/Patients.js [hunk 6]
```diff
@@ -4,18 +4,33 @@ import PatientItem from './PatientItem'
...
         loading: true,
       };
-
       navigation = this.props.navigation;
+      const { uid } = Firebase.auth.currentUser;
+      Firebase.firestore.collection("users").doc(uid).get().then(
+        (doc)=>{
+          data=doc.data()
+          this.setState({role: data.role})
+        //console.log(data);
```
Comment (Danny393) [inline src/vet/patients/Patients.js]: Is this comment still needed?

File: src/components/Theme.js [hunk 7]
```diff
@@ -46,20 +46,20 @@ type Theme = {
 
 const theme: Theme = {
     palette: {
-        primary: "#81f1f7",
+        primary: "#39c3ef",
```
Comment (Danny393) [inline src/components/Theme.js]: I think I prefer the old color, but we can ask the team on Monday what they think.

**Review by Danny393 — CHANGES_REQUESTED** [review]
I'm not sure if you forgot to push any final changes to this PR, if so then next time make sure to only create a PR when you are 100% ready for it to be merged into main with no further changes.
I merged main into your branch so everything else is now up to date, make sure to pull before making any changes.

**Comment by OAKevin** [comment]
fixed the spacing, reverted the Button and Fields back to mains version

**Pull request merged**